whylog/client/__init__.py

```diff
@@ -1,6 +1,21 @@
-class WhylogClient(object):
-    def __init__(self, **kwargs):
+from abc import ABCMeta, abstractmethod
+
+
+class AbstractClient(object):
+
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def get_cause(self):
+        pass
+
+
+class WhylogClient(AbstractClient):
+    def __init__(self, config):
+        self.config = config
+
+    def get_cause(self, offset, vim_line):
```

Comment:
`vim_line` is too specific. I think it was supposed to be called `front_input` - please confirm with Andrzej.